### PR TITLE
Fix : pictures of de- and rescheduled stories do not get published [SDCP-686]

### PR DESCRIPTION
**Pull request by devketanpro**

### CODE REVIEW

File: apps/archive/archive.py [hunk 1]
```diff
@@ -870,8 +870,6 @@ def deschedule_item(self, updates, original):
         :param original: original is document.
         """
         updates[ITEM_STATE] = CONTENT_STATE.PROGRESS
-        updates[PUBLISH_SCHEDULE] = original[PUBLISH_SCHEDULE]
-        updates[SCHEDULE_SETTINGS] = original[SCHEDULE_SETTINGS]
```
Comment (petrjasek) [inline apps/archive/archive.py:874]: I think we want to unset it, but only if that item was auto published

Comment (devketanpro) [reply apps/archive/archive.py:874]: I see, but how can be identify item auto published.

Comment (petrjasek) [reply apps/archive/archive.py:874]: I think we set `auto_publish=True` on those

Comment (devketanpro) [reply apps/archive/archive.py:874]: okay let me check.

Comment (petrjasek) [reply apps/archive/archive.py:874]: I think you have to change it not only on the picture item itself but also in the associations of the text item

**Pull request merged**